content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -72,16 +72,36 @@ The upgrade workflow at high level is the following:
 <!--
 ### Additional information
 
-- [Draining nodes](/docs/tasks/administer-cluster/safely-drain-node/) before kubelet MINOR version
-  upgrades is required. In the case of control plane nodes, they could be running CoreDNS Pods or other critical workloads.
+- The instructions below outline when to drain each node during the upgrade process.
+If you are performing a **minor** version upgrade for any kubelet, you **must**
+first drain the node (or nodes) that you are upgrading. In the case of control plane nodes,
+they could be running CoreDNS Pods or other critical workloads. For more information see
+[Draining nodes](/docs/tasks/administer-cluster/safely-drain-node/).
 - All containers are restarted after upgrade, because the container spec hash value is changed.
 -->
+
 ### 附加信息
 
-- 在对 kubelet 作次版本升版时需要[腾空节点](/zh/docs/tasks/administer-cluster/safely-drain-node/)。
-  对于控制面节点，其上可能运行着 CoreDNS Pods 或者其它非常重要的负载。
+- 下述说明了在升级过程中何时腾空每个节点。如果你正在对任何 kubelet 进行小版本升级，
+  你需要先腾空待升级的节点（或多个节点）。对于控制面节点，其上可能运行着 CoreDNS Pods
+  或者其它非常重要的负载。更多信息见 [腾空节点](/zh/docs/tasks/administer-cluster/safely-drain-node/)。
```

Comment:
```suggestion
  或者其它非常重要的负载。更多信息见[腾空节点](/zh/docs/tasks/administer-cluster/safely-drain-node/)。
```

Reply:
There are [some people](https://github.com/sparanoid/chinese-copywriting-guidelines#%E7%88%AD%E8%AD%B0) think there should a space between link and text. And I personally prefer this too.

(It's not mentioned in [k8s zh l10n gulidline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E5%85%B3%E4%BA%8E%E9%93%BE%E6%8E%A5).)

Reply:
I think we only keep space between English and Chinese, so I have remove this space.